src/components/Company.js

```diff
@@ -1,20 +1,21 @@
 import React, { Fragment } from 'react';
 import Header from './Header';
-import '../styles/Company.scss';
+import { CategorySection } from './CategorySections';
 
 function CompanyInfo (props) {
-  return (
-    <Fragment>
+  const { companyData } = props;
+  const { effectOnEcosystem, politicalContributions, laborPractices, lawsuits, demographics } = companyData;
+  console.log(companyData);
+  return companyData ? (
```

Comment:
I'm using a ternary here `companyData ? <components> : <fallback message>`. This means if there is company data, return what's after the question mark, if not, return what's after the colon.